### ISSUE TITLE
Setting the param stopbackward=1 means fine-tuning ?

### ISSUE THREAD

**Issue by lustory** [AlexeyAB/darknet#4591]
In the "How to improve object detection" section, it says "do Fine-Tuning instead of Transfer-Learning, set param stopbackward=1".  Is this description accurate? I do not doubt this process will speed up the training process but the statement of the this sentence. Set stopbackward = 1 means to stop the weights update of the CNN, this implies to perfom fine-tuning or transfer learning?

**Comment by AlexeyAB** [comment]
Full quote: https://github.com/AlexeyAB/darknet#how-to-improve-object-detection

> to speedup training (with decreasing detection accuracy) do Fine-Tuning instead of Transfer-Learning, set param stopbackward=1 here: https://github.com/AlexeyAB/darknet/blob/6d44529cf93211c319813c90e0c1adb34426abe5/cfg/yolov3.cfg#L548 then do this command: ./darknet partial cfg/yolov3.cfg yolov3.weights yolov3.conv.81 81 will be created file **yolov3.conv.81**, then train by using weights file yolov3.conv.81 instead of darknet53.conv.74

Using `yolov3.conv.81` pre-trained weights **for training Detector** means Fine-Tuning.

Read more: https://github.com/AlexeyAB/darknet/issues/2139

**Comment by akashAD98** [comment]
@AlexeyAB  so it's also called training from scratch??
